missile-strike.yaml

```diff
@@ -0,0 +1,23 @@
+Reference: "Some locked reference"
+title: "Example Air Missile Strike Operation"
+startDate: "2024-03-01T00:00:00Z"
+endDate: "2024-03-02T00:00:00Z"
+activity: "kinetic"
+activityDescription: "A kinetic missile strike targeting strategic assets."
+intent: "Neutralize high-value targets to degrade enemy capabilities."
+actionsOn: "Immediate action upon confirmation of target location."
+synchronisedWith: ["o-a", "o-b"]
+supports: "Supports main operation by degrading enemy air defense."
+activityCard: "FR Cyper Op 12 - Green Drain"
+rulesOfEngagement: "Weapons Free"
+emconState: "Electronic Silence"
+requiresAirToAir: false
+ownAssets:
+  - asset: "ta"
+    missileType: "SRBM"
+    number: 2
+otherAssets:
```

Comment:
This is good work, but could you create multiple entries to ownAssets and otherAssets , so show it´s an array?

Reply:
Okay, No problem

Reply:
I've done that